mfhdf/src/nc_priv.h

```diff
@@ -45,9 +45,6 @@
 #define NC_DOUBLE_SIZE      8
 #define NC_UNSPECIFIED_SIZE 0
 
-/* ptr argument type in internal functions */
-#define Void char
```

Comment:
WOW

Reply:
HDF4 is always a journey down memory lane

Reply:
At least this one is internal. The VOID macro is in the public API and HDF-EOS2 uses it.